src/concepts/segments.md

```diff
@@ -105,7 +104,7 @@ So if a user was active as a regular user v3 on e.g. the second day of a 28-day
 
 Since many user states use the full extent of the `*_bits` column wizardry in `clients_last_seen`, you'll have to query WAU or MAU the old fashioned way:
 
-```lang=sql
+```sql
```

Comment:
Note that SQL code blocks need to be declared this way in order for the syntax highlighter to pick them up.

Reply:
> Note that SQL code blocks need to be declared this way in order for the syntax highlighter to pick them up.

Reply:
> > Note that SQL code blocks need to be declared this way in order for the syntax highlighter to pick them up.
> 
>